plugins/env/payload-schema.yml

```diff
@@ -0,0 +1,6 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "EnvPayload"
+type:               "object"
+patternProperties:
+  .+:
+    type: string
```

Comment:
Don't use pattern properties...
Just use `additionalProperties: {type: string}`

Also add some "description" properties, ideally with a description that can serve as documentation... It doesn't have to be long, but a line, two or 5 would be good. Usually using markdown in description properties is fine, we do it everywhere...